### ISSUE TITLE
"run_dien" leads to ValueError for 'seq_length'

### ISSUE THREAD

**Issue by Curlykonda** [shenweichen/DeepCTR#186]
**Describe the bug(问题描述)**
DIEN Model cannot be built because "seq_length" is used 2 times

`line 278, in DIEN
   model = tf.keras.models.Model(inputs=model_input_list, outputs=output)`
[...]
`ValueError: The name "seq_length" is used 2 times in the model. All layer names should be unique.`

**To Reproduce(复现步骤)**
1. Go to 'examples'
2. Run 'run_dien.py'
3. See error when attempting to build `Model` with `model_input_list`

**Operating environment(运行环境):**
 - python version [3.6]
 - tensorflow version [2.1.0]
 - deepctr version [0.7.0]

**Additional context**
In an attempt to solve this issue, I've found that the order of items in `model_input_list` matters. E.g. when deleting the item corresponding to `seq_length` and only adding it via `model_input_list += [user_behavior_length]` (so that it's the last item in the list), this error does NOT occur. While this approach allows the example to run further, the implementation is sub-optimal. Moreover, overcoming this issue subsequently leads to another issue (as pointed out in #160 ).

**Comment by e199612** [comment]
I have the same problem.
Did you solve it?

Thanks

**Comment by Curlykonda** [comment]
As a temporary solution, one could simply remove the item from the list and add it again, so that it becomes the last element. 

```
if str(model_input_list[6].name).__contains__('seq_len'):
        del(model_input_list[6])
        model_input_list += [user_behavior_length]

model = tf.keras.models.Model(inputs=model_input_list, outputs=output)
```

**Comment by shenweichen** [comment]
hi we have solved this problem in deepctr v0.8.1,please use `pip install -U deepctr` to upgrade your version.
